.editorconfig

```diff
@@ -0,0 +1,8 @@
+# Override for Makefile
+[{Makefile, makefile, GNUmakefile}]
+indent_style = tab
+indent_size = 4
+
+[Makefile.*]
+indent_style = tab
+indent_size = 4
```

Comment:
Do we also want to add a section for `*.tf` ?

Reply:
Good catch!